### PR TITLE
Wire in blob validation

### PR DESCRIPTION
**Pull request by henridf**

### CODE REVIEW

**Comment by tersec** [comment]
Minimal-preset Deneb/Cancun engine API still isn't a thing.

File: beacon_chain/gossip_processing/block_processor.nim [hunk 1]
```diff
@@ -195,8 +197,19 @@ proc storeBackfillBlock(
...
+                                 blobs.mapIt(it.kzg_proof))
+          if r.isErr():
+            debug "backfill blob validation failed",
+             blockRoot = shortLog(signedBlock.root),
+             blck = shortLog(signedBlock.message),
+             signature = shortLog(signedBlock.signature),
+             msg = r.error()
+          r.isOk()
+        else:
+            true
```
Comment (etan-status) [inline beacon_chain/gossip_processing/block_processor.nim:214]:
```suggestion
        else:
          true
```

**Comment by henridf** [comment]
Latest failure seems to be spurious:

> /github-runner/workspace/nimbus-eth2/config.nims(13, 11) Error: cannot open file: nimbus-build-system.paths

**Comment by tersec** [comment]
> Latest failure seems to be spurious:
> 
> > /github-runner/workspace/nimbus-eth2/config.nims(13, 11) Error: cannot open file: nimbus-build-system.paths

Agree, it's spurious. cc @zah

**Review by tersec — APPROVED** [review]

**Pull request merged**